src/Tribe/PUE/Checker.php

```diff
@@ -467,68 +467,50 @@ public function ajax_validate_key() {
 				'<a href="https://theeventscalendar.com/license-keys/">',
 				'</a>'
 			);
 
-			if ( ! empty( $info->api_invalid_message ) ) {
-				$message = wp_kses( $info->api_invalid_message, 'post' );
+			if ( ! empty( $pluginInfo->api_invalid_message ) ) {
+				$message = wp_kses( $pluginInfo->api_invalid_message, 'post' );
 			}
 
 			$message = str_replace( '%plugin_name%', '<b>' . $this->get_plugin_name() . '</b>', $message );
 			$message = str_replace( '%plugin_slug%', $this->get_slug(), $message );
 			$message = str_replace( '%update_url%', $this->get_pue_update_url(), $message );
-			$message = str_replace( '%version%', $info->version, $message );
+			$message = str_replace( '%version%', $pluginInfo->version, $message );
 
 			return $message;
 		}
 
 		/**
-		 * Echo JSON formatted errors
+		 * Displays an error notice if a premium plugin is activated and the license is expired
+		 *
+		 * @since 4.3
 		 */
-		public function display_json_error() {
-			$pluginInfo       = $this->json_error;
-			$update_dismissed = $this->get_option( $this->dismiss_upgrade );
-
-			$is_dismissed = ! empty( $update_dismissed ) && in_array( $pluginInfo->version, $update_dismissed ) ? true : false;
-
-			if ( $is_dismissed ) {
-				return;
-			}
+		public function display_expired_license_message() {
+			$pluginInfo  = $this->json_error;
 
 			if ( ! current_user_can( 'administrator' ) ) {
 				return;
 			}
+			$expired_license_msg     = $this->get_api_message( $pluginInfo );
+			$expired_license_message = str_replace( '%plugin_name%', '<strong>' . $this->get_plugin_name() . '</strong>', $expired_license_msg );
+			?>
+			<div class="notice notice-warning is-dismissible" id="pu-dashboard-message">
+				<div class="tribe-message">
+					<img class="spirit-animal" src="<?php esc_html_e( plugins_url( '../../src/resources/images/tec-panda.png', dirname( __FILE__ ) ) );?>">
 
-			//only display messages if there is a new version of the plugin.
-			if ( version_compare( $pluginInfo->version, $this->get_installed_version(), '>' ) ) {
-				if ( empty( $pluginInfo->api_invalid ) || $pluginInfo->api_invalid != 1 ) {
-					return;
-				}
-
-				$msg = $this->get_api_message( $pluginInfo );
-
-				//Dismiss code idea below is obtained from the Gravity Forms Plugin by rocketgenius.com
-				?>
-				<div class="updated" style="padding:5px; position:relative;" id="pu_dashboard_message"><?php echo wp_kses( $msg, 'post' ); ?>
-					<a href="javascript:void(0);" onclick="PUDismissUpgrade();" style="float:right;">[X]</a>
+					<p><?php echo wp_kses( $expired_license_message, 'post' ); ?></p>
+					<p><a href="https://theeventscalendar.com/my-account/"><?php esc_html_e( 'Renew your license', 'tribe-common' ); ?></a> <?php esc_html_e( 'to get access to the latest versions including bug fixes, security updates, and new features.', 'tribe-common' ); ?></p>
```

Comment:
We talked about this on the call the other day, too, and I think I misunderstood your original question now that this code is staring at my face for a review. Forgive me :) Let's keep all the surrounding HTML separate (as you have it) except for one piece - the message itself. Rather than breaking up the message to accommodate the anchor tag, let's inject the anchor tag using `printf`:

```php
<p>
  <?php
    printf(
      esc_html__(
        '%1$sRenew your license%2$s to get access to the latest versions including bug fixes, security updates, and new features.',
        'tribe-common'
      ),
      '<a href="https://theeventscalendar.com/my-account/">',
      '</a>'
    );
  ?>
</p>
```